### DISCUSSION TITLE
Can u give some toy data demo to run training scripts?

### DISCUSSION THREAD

**Discussion by xiao2mo** [PixArt-alpha/PixArt-alpha#19] [General]
Thanks a lot.

**Comment by yujincheng08** [comment]
if u need a quick trial, try to use the diffusers pipeline for training.

**Reply by wen020** [reply]
I use diffusers pipeline training and use pixart as the base model(https://huggingface.co/PixArt-alpha/PixArt-XL-2-512x512), but an error will be reported.
![image](https://github.com/PixArt-alpha/PixArt-alpha/assets/54690997/b4fb6cad-912f-4eba-afd4-87c09ae340ec)

And the base model have not find unet.

Please tell me how to solve this. thanks

**Reply by lawrence-cj** [reply]
Please open an issue to diffusers for help. I see that you have already open an issue on [Hugging Face](https://github.com/huggingface/diffusers/issues/5777)

**Reply by storuky** [reply]
@lawrence-cj but you have examples of pixart-alpha+dreambooth. Any chance you can share sources of how these models were trained using dreambooth?

**Reply by yujincheng08** [reply]
all u need is to change the dataloader and hyperparams.

**Reply by lawrence-cj** [reply]
Maybe we can release the dataloader soon. @yujincheng08

**Reply by kopyl** [reply]
Why you can't load a dataset from Hugginface? In the same format which would be needed for SD 1.5 fine-tune.

**Reply by lawrence-cj** [reply]
May I have a doc for what u said so that I can follow up?

**Reply by kopyl** [reply]
@lawrence-cj are you replying to me?

Check this:
https://github.com/huggingface/diffusers/tree/main/examples/text_to_image

You can train a model like this:

```
export MODEL_NAME="CompVis/stable-diffusion-v1-4"
export DATASET_NAME="lambdalabs/pokemon-blip-captions"

accelerate launch --mixed_precision="fp16"  train_text_to_image.py \
  --pretrained_model_name_or_path=$MODEL_NAME \
  --dataset_name=$DATASET_NAME \
  --use_ema \
  --resolution=512 --center_crop --random_flip \
  --train_batch_size=1 \
  --gradient_accumulation_steps=4 \
  --gradient_checkpointing \
  --max_train_steps=15000 \
  --learning_rate=1e-05 \
  --max_grad_norm=1 \
  --lr_scheduler="constant" --lr_warmup_steps=0 \
  --output_dir="sd-pokemon-model" 
```

with a dataset created like this:
https://huggingface.co/docs/datasets/image_dataset

Which looks like this (my dataset): https://huggingface.co/datasets/kopyl/833-icons-dataset-1024-blip-large

**Reply by lawrence-cj** [reply]
> @lawrence-cj are you replying to me?

Oh yah, I thought you are talking to me. lol

**Reply by kopyl** [reply]
@lawrence-cj yeah, to you. But I'm not sure who are you replying to without mentioning anyone.

**Comment by kopyl** [comment]
@wen020 i also want to try to fine-tune the 512 model

**Comment by lawrence-cj** [comment]
Toy images added. closed

**Reply by kopyl** [reply]
@lawrence-cj thank you very much :)
